include/ASSFunctions.h

```diff
@@ -1,9 +1,19 @@
 #include <vector>
 #include "vex.h"
 
+//Begins Automated Steering System
+void ASSInit();
+
+//Returns a vector representing a 360 degree scan
 std::vector<bool> scan();
 
-void turnToPath(std::vector<bool> data);
+//Picks an angle of next path based on scan data parameter
+//Param degreesOfScanSeparation refers to the gap between individual scans
+double findTurnAngle(std::vector<bool> data, double degreesOfScanSeparation);
+
+//Turns the robot to the specified angle
+void turnToAngle(double angle);
 
+//Drives the robot forward until an object is detected in front
 void driveUntilWall();
```

Comment:
not sure how I feel about this abstraction using a .h file. from my research, a .h file is only meant to be used as a _public interface_ (see [here](https://www.google.com/search?q=should+a+header+file+just+be+a+public+interface&rlz=1C1CHBF_enUS773US773&oq=should+a+header+file+just+be+a+public+interface&aqs=chrome..69i57j33i160.7826j1j7&sourceid=chrome&ie=UTF-8)). the only function I see being used publicly is ASSInit in main.cpp, meaning I believe there are redundant declarations in this .h file. this would mean that a fix would be to make the only function in the .h file be ASSInit. Not sure if we should still use a .h file for one function but I do think that seems to be the way you're supposed to do it... see [here](https://riptutorial.com/c/example/3250/calling-a-function-from-another-c-file).

Reply:
I just changed ASS.h to only include ASSInit()